include/qengine_cpu.hpp

```diff
@@ -38,20 +38,26 @@ class QEngineCPU : public QEngine, public ParallelFor {
 
 public:
     QEngineCPU(bitLenInt qBitCount, bitCapInt initState, std::shared_ptr<std::default_random_engine> rgp = nullptr,
-        complex phaseFac = complex(-999.0, -999.0), bool doNorm = true, bool useHostMem = false);
+        complex phaseFac = complex(-999.0, -999.0), bool doNorm = true, bool randomGlobalPhase = true,
+        bool useHostMem = false);
```

Comment:
bool ignored = false); ... and in the constructor `: useHostRam(true), ...` since it can never be false for QEngineCPU.

Reply:
Done.